about/policy/metadata/index.md

```diff
@@ -6,109 +6,57 @@ tags:
 
 All words in tags should be title-cased except for articles (“of”, “the”, “and”, etc.) or abbreviations. Each tag should be separated by a comma. Please refer to [Using tags effectively](https://www.esri.com/arcgis-blog/products/arcgis-online/data-management/using-tags-effectively/) for more ideas and information.
 
-`example: AGRC, SGID, Recreation, Paths, Biking, Hiking, Horseback Riding`
+Example: `AGRC, SGID, Recreation, Paths, Biking, Hiking, Horseback Riding`
 
 #### Summary
 {: .text-left}
 
-The summary should summarize the description in approximately 2 to 3 sentences.
+The summary should _briefly_ explain the dataset in under 250 characters (an ArcGIS Online limitation). It is primarily used when reviewing ArcGIS Online search results. The summary may end up being very similar to the dataset name for some datasets.
 
-The summary should address the questions presented above in the [summarize your data](#summarize-your-data) section.
+Example: `Utah public library and bookmobile locations`
 
 #### Description
 {: .text-left}
 
-The Description is a more in-depth version of the summary and it should stand on its own. The description should expand on the summary. This is important, particularly, in the event that the summary is missing in the metadata (which is currently the case with [Open Data](https://opendata.gis.utah.gov/) datasets).
-
-The first line of the description should be a date, indicating the last update.
+The description is a more in-depth version of the summary and should stand on its own. It is a great place for details on the source of the data, how frequently it's updated, its usability at certain scales, or any common misconceptions. A good description is also important because it's the main info displayed in [Open Data](https://opendata.gis.utah.gov/) (the summary is not shown).
 
-This section should also contain a link to the corresponding [gis.utah.gov data page]({% link data/index.html %})
+The first line of the description should be be the date the dataset was last updated so people have a clear and consistent place to see how fresh or stale it is.
 
-Additionally, the description should address the questions presented above in the [describe your data](#describe-your-data) section.
+For datasets hosted by AGRC, this section should also contain a link to the corresponding [gis.utah.gov data page]({% link data/index.html %}).
 
 #### Credits (as in the source of the data)
 {: .text-left}
 
-This should cover who created the dataset and how you can get in contact with them.
+The credits should clearly state who created the dataset and how you can get in contact with them.
 
-The credits should address the questions presented above in the [establish contact info](#establish-contact-info) section.
+Example: `AGRC, UDOT`
```

Comment:
**question**: Should this example include any contact information? The description above references contact information.